pkg/asset/ignition/bootstrap/bootstrap.go

```diff
@@ -208,11 +210,25 @@ func (a *Bootstrap) getTemplateData(installConfig *types.InstallConfig, proxy *c
 		logrus.Debugf("Using internal constant for release image %s", releaseImage)
 	}
 
+	var rhcosImageBaseURI string
+	if ri, ok := os.LookupEnv("OPENSHIFT_INSTALL_IMAGE_BASE_URI_OVERRIDE"); ok && ri != "" {
```

Comment:
@derekhiggins I think it would perhaps be better if we used this override instead of CACHEURL, because it's one less thing we'll need to hard-code (and ultimately template) in startironic.sh - what are your thoughts on instead making the https://github.com/openshift-metal3/rhcos-downloader/blob/master/get-resource.sh output similar to the original repo structure, or we can cache differently in dev-scripts, such that we just set OPENSHIFT_INSTALL_IMAGE_BASE_URI_OVERRIDE in dev-scripts?

Reply:
Cancel that, @abhinavdahiya has indicated in https://github.com/openshift/installer/pull/2035 that we shouldn't add a new variable like this